modules/data-entry/src/main/frontend/src/dataHomepage/ExportButton.jsx

```diff
@@ -210,6 +289,28 @@ function ExportButton(props) {
                 { entitySpecs[type].avatar ? <Icon>{entitySpecs[type].avatar}</Icon> : type?.charAt(0) }
             </Avatar>);
   }
+
+  let getDatePicker = (value, setter, label) => {
+    return (<DatePicker            
```

Comment:
Unnecessary trailing whitespace:
```suggestion
    return (<DatePicker
```